src/styles/bannerCommon.ts

```diff
@@ -238,3 +238,261 @@ export const styles = {
         }
     `,
 };
+
+interface StyleData {
```

Comment:
I don't _think_ these are used anywhere other than the new banner component currently, so I'd vote to colocate them with that code, rather than putting them in bannerCommon. What do you think?

Reply:
My thinking is that as soon as we deploy this, Marketing will come back and ask us for a similarly configurable newsletter banner. Both the existing NewsletterBanner and BannerWithLink use the exported `style` object on this page. A new StyleableNewsletterBanner could make use of the `selfServeStyles` function ... so my gut instinct is to leave the function here. I know it's future-proofing for something that may not happen - but does that make sense?